Travis

```
[ant:checkstyle] [ERROR] /home/travis/build/spring-projects/spring-integration/spring-integration-http/src/main/java/org/springframework/integration/http/support/IntegrationWebExchangeBindException.java:19:8: Unused import - org.springframework.core.MethodParameter. [UnusedImports]
[ant:checkstyle] [ERROR] /home/travis/build/spring-projects/spring-integration/spring-integration-http/src/main/java/org/springframework/integration/http/support/IntegrationWebExchangeBindException.java:20:8: Unused import - org.springframework.messaging.Message. [UnusedImports]
[ant:checkstyle] [ERROR] /home/travis/build/spring-projects/spring-integration/spring-integration-http/src/main/java/org/springframework/integration/http/support/IntegrationWebExchangeBindException.java:21:8: Unused import - org.springframework.util.Assert. [UnusedImports]
```

--- a/spring-integration-http/src/main/java/org/springframework/integration/http/support/IntegrationWebExchangeBindException.java
+++ b/spring-integration-http/src/main/java/org/springframework/integration/http/support/IntegrationWebExchangeBindException.java
@@ -16,9 +16,6 @@
 
 package org.springframework.integration.http.support;
 
-import org.springframework.core.MethodParameter;
-import org.springframework.messaging.Message;
-import org.springframework.util.Assert;
 import org.springframework.validation.BindingResult;
 import org.springframework.validation.ObjectError;
 import org.springframework.web.bind.support.WebExchangeBindException;

